includes/WP_Admin.php

```diff
@@ -106,6 +106,9 @@ public static function register_assets() {
 
 			\wp_enqueue_script( self::$slug );
 			\wp_enqueue_style( self::$slug );
+
+			// This hook exists in the patterns module.
+			\do_action( 'enqueue_nfd_wonder_blocks_utilities' );
```

Comment:
This hook was added [here](https://github.com/newfold-labs/wp-module-patterns/pull/5)